pyproject.toml

```diff
@@ -111,6 +111,9 @@ docs = [
     "pylint", # Generates UML diagrams
     "Jinja2", # Used in numpydoc and nbconvert
     "sphinxcontrib-jquery==4.1", # Handle missing jquery errors
+    "sphinx-simplepdf==1.6.0", # Used to make PDF versions of the docs
+    "jupyter-contrib-nbextensions", # A collections of JS extensions for jupyter notebooks
+    "lxml<5.0.0", # Needed because the dep above is no longer an active project
```

Comment:
Merge conflict error! I think you want these deleted

Reply:
Sorry, yup. Fixed it while you were writing that. My bad.